src/Analyzers/CSharp/Tests/ImplementInterface/ImplementInterfaceTests.cs

```diff
@@ -4663,81 +4663,87 @@ public async Task TestOptionalDateTime1()
 
-            interface IGoo
-            {
-                void Goo([Optional][DateTimeConstant(100)] DateTime x);
-            }
+                interface IGoo
+                {
+                    void Goo([Optional][DateTimeConstant(100)] DateTime x);
+                }
 
-            public class C : IGoo
-            {
-                public void Goo([DateTimeConstant(100), Optional] DateTime x)
+                public class C : IGoo
                 {
-                    throw new NotImplementedException();
+                    public void Goo([DateTimeConstant(100), Optional] DateTime x)
+                    {
+                        throw new NotImplementedException();
+                    }
                 }
-            }
-            """,
+                """,
             Options = { AllOptionsOff },
 
-            // 🐛 one value is generated with 0L instead of 0
+            // 🐛 one value is generated with 100L instead of 100L
             CodeActionValidationMode = CodeActionValidationMode.None,
         }.RunAsync();
     }
 
     [Fact, WorkItem("http://vstfdevdiv:8080/DevDiv2/DevDiv/_workitems/edit/545476")]
     public async Task TestOptionalDateTime2()
     {
-        await TestWithAllCodeStyleOptionsOffAsync(
-            """
-            using System;
-            using System.Runtime.CompilerServices;
-            using System.Runtime.InteropServices;
+        await new VerifyCS.Test
+        {
+            TestCode = """
+                using System;
+                using System.Runtime.CompilerServices;
+                using System.Runtime.InteropServices;
 
-            interface IGoo
-            {
-                void Goo([Optional][DateTimeConstant(100)] DateTime x);
-            }
+                interface IGoo
+                {
+                    void Goo([Optional][DateTimeConstant(100)] DateTime x);
+                }
 
-            public class C : {|CS0535:IGoo|}
-            {
-            }
-            """,
-            """
-            using System;
-            using System.Runtime.CompilerServices;
-            using System.Runtime.InteropServices;
+                public class C : {|CS0535:IGoo|}
+                {
+                }
+                """,
+            FixedCode = """
+                using System;
+                using System.Runtime.CompilerServices;
+                using System.Runtime.InteropServices;
 
-            interface IGoo
-            {
-                void Goo([Optional][DateTimeConstant(100)] DateTime x);
-            }
+                interface IGoo
+                {
+                    void Goo([Optional][DateTimeConstant(100)] DateTime x);
+                }
 
-            public class C : IGoo
-            {
-                void IGoo.Goo(DateTime x)
+                public class C : IGoo
                 {
-                    throw new NotImplementedException();
+                    void IGoo.Goo([DateTimeConstant(100), Optional] DateTime x)
```

Comment:
these attributes seem undesirable.

Reply:
that doesn't help here.  i don't see how DateTimeConstant *or* optional are valuable here.

Reply:
Removed the attributes for the reason mentioned in the comment above